code/__DEFINES/maths.dm

```diff
@@ -84,9 +84,11 @@
 
 #define ISABOUTEQUAL(a, b, deviation) (deviation ? abs((a) - (b)) <= deviation : abs((a) - (b)) <= 0.1)
 
-#define ISEVEN(x) (x % 2 == 0)
+// Note for even and odd:
+// you can get the same result with 'x % 2 == 0', but calculation(divide by 2) is more expensive than comparison. We don't have to calculate when we know first bitflag determines if a value is odd or even.
```

Comment:
This comment is wrong.

Division is extremely cheap. What isn't extremely cheap is finding the remainder of division. Secondly, you aren't comparing, you're doing a bitwise math operation, comaprison is `==`. Bitwise math is actually really slow in byond due to how numbers work in the engine, but it is funny it's slower than modulo.

Reply:
Oh, thank you for correcting the misunderstanding. I will fix the comment